src/components/molecules/UiPhoneNumber/UiPhoneNumber.vue

```diff
@@ -105,6 +103,13 @@ const modelValue = computed({
   get: () => props.modelValue,
   set: (value) => emit('update:modelValue', value),
 });
+const attrs = useAttrs() as { class:string };
```

Comment:
Why do you do `as { class:string };` here?

Reply:
Because of ts return `error` for class in `attrs`. You can see it now.